packages/components/src/button/index.tsx

```diff
@@ -110,6 +110,7 @@ export function UnforwardedButton(
 		children,
 		size = 'default',
 		text,
+		tooltipDirection,
```

Comment:
Note that there is already a `tooltipPosition` prop on `Button` that can you can use to change the tooltip placement (e.g. `"top"` and `"bottom"`).

I don't think we will need any changes in the `@wordpress/components` package to land this PR. As for fine-tuning the "bottom" placement so the block mover down tooltip aligns with the other toolbar button tooltips, I think a cleaner strategy would be to fix the metrics of the block mover buttons so their vertical edges are aligned with the other buttons. If you add a visible outline to the buttons, you can see that the block mover buttons are shorter. This is causing the misalignment of the tooltips.

<img width="310" alt="Toolbar button metrics" src="https://github.com/user-attachments/assets/62e6bf52-fa71-4f4a-96d7-0d04b077fb60">

Reply:
Hi @mirka, Agree on this, We just need to adjust this style added here which make this height - https://github.com/WordPress/gutenberg/blob/46f9fe0f31bf9ce5f6210cad96074206e90527e6/packages/block-editor/src/components/block-mover/style.scss#L12 as `24px`, by removing the `$grid-unit-05`, not sure why this added, may be to decrease the space between chevrons,

Also, need to adjust the top and bottom padding for `up` and `down` button to `10px`, so that it looks what it looked like earlier.

<img width="1440" alt="Screenshot 2024-08-14 at 10 14 22 AM" src="https://github.com/user-attachments/assets/ad0a3f8b-957f-441d-ab73-6e197ed1230d">

@Rishit30G, Could you please adjust this style and check if it helps, 

The `move up`, would still not in place, but `move down` would be consistent with different toolbar tooltips.

Reply:
Agreed @mirka's suggestions are the way to go. We should use what we have and keep any specific customization outside of the `Tooltip` component.

Reply:
Thanks for the suggestions @mirka, @hbhalodia, @tyxla 🙇 
- I have removed extra `tooltipDirection` and used `tooltipPosition` 
- Updated the style from `height: $block-toolbar-height * 0.5 - $grid-unit-05;` to `height: $block-toolbar-height * 0.5;`

Reply:
> the "$grid-unit-05", not sure why this added

This style was added when introducing a custom scrollbar to the block toolbar in #57444 (Sorry for not leaving a comment 🙇‍♂️).

If we remove `$grid-unit-05`, an unnatural shift will occur when the mover button is focused on Windows OS:

https://github.com/user-attachments/assets/c6e862f4-a3d7-45dd-8a5f-55966b6a920f

We tried a lot of different things to fix this problem in #57444, but for now, I think it's better not to remove `$grid-unit-05`.

Of course, if the top toolbar is disabled, there's no problem with removing `$grid-unit-05`, but if adjusting the tooltip position becomes complicated, I think it's better to leave `$grid-unit-05`.

Reply:
Thanks for the info @t-hamano! 🙌🏻
I already had a workaround for this in a previous commit, will see if I can incorporate it

Reply:
Fixed ✅

Reply:
Hey @Rishit30G , I agree with @mirka 's suggestions above, especially about

> I don't think we will need any changes in the `@wordpress/components` package to land this PR

Can we make sure this PR doesn't include changes to `Button` and `Tooltip` ? Thank you 🙏